pkg/transform/reportoutput/json.go

```diff
@@ -0,0 +1,87 @@
+package reportoutput
+
+import (
+	"encoding/json"
+
+	"github.com/fusor/cpma/pkg/io"
+	"github.com/pkg/errors"
+)
+
+func jsonOutput(r ReportOutput) error {
+	var existingReports ReportOutput
+
+	jsonFile := "report.json"
```

Comment:
This function code was copied from old `DumpReport()` function.  I see places for improvements here, just need some time for it.